roboteam_ai/src/skills/GoToPos.cpp

```diff
@@ -35,7 +35,11 @@ bt::Node::Status GoToPos::onUpdate() {
     }
 
     control::PosVelAngle pva = gotopos.getPosVelAngle(robot, targetPos);
-    pva.vel = control::ControlUtils::velocityLimiter(pva.vel, maxVel);
+    pva.vel = control::ControlUtils::velocityLimiter(pva.vel, maxVel, minVel);
```

Comment:
I think it would be nice if all these control calls/ad hoc changes to commands can be in one function call that we can easily implement in skill or such. For now this is fine but if we want to change it structurally this needs to be changed structurally